deployments/hadoop-yarn/ansible/29-install-pip-libs.yml

```diff
@@ -27,9 +27,19 @@
     - config/ansible.yml
     - /tmp/ansible-vars.yml
 
+
+
   tasks:
 
-    - name: "Install Python libraries"
+    - name: Copy pip requirements file into tmp
+      become: true
+      copy:
+        src: "{{ playbook_dir | dirname | dirname }}/common/pip/requirements.txt"
+        dest: "/tmp/requirements.txt"
+
+
+    - name: Install the required Python packages
       become: true
-      command: /usr/bin/pip3 install numpy scipy matplotlib grpcio jupyter cython protobuf pandas healpy astropy scikit-learn hdbscan pyvo astroquery
```

Comment:
hasn't this already been done in PR #463 ?

Reply:
Can we confirm that this is no longer part of the pull request ?
If merging master back in to this branch resolved it, then why is this still showing up in the  pull request as a change for review ?